### ISSUE TITLE
strange psutils failures

### ISSUE THREAD

**Issue by rokroskar** [SwissDataScienceCenter/renku-jupyter#14]
For example, in the docker build: 

```
    building 'psutil._psutil_linux' extension
    creating build/temp.linux-x86_64-3.6
    creating build/temp.linux-x86_64-3.6/psutil
    x86_64-conda_cos6-linux-gnu-gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -fwrapv -O2 -Wall -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -pipe -fdebug-prefix-map=/home/conda/feedstock_root/build_artifacts/python_1551150806669/work=/usr/local/src/conda/python-3.6.7 -fdebug-prefix-map=/opt/conda/envs/renku=/usr/local/src/conda-prefix -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -pipe -fdebug-prefix-map=/home/conda/feedstock_root/build_artifacts/python_1551150806669/work=/usr/local/src/conda/python-3.6.7 -fdebug-prefix-map=/opt/conda/envs/renku=/usr/local/src/conda-prefix -fPIC -DPSUTIL_POSIX=1 -DPSUTIL_VERSION=551 -DPSUTIL_LINUX=1 -I/opt/conda/envs/renku/include/python3.6m -c psutil/_psutil_common.c -o build/temp.linux-x86_64-3.6/psutil/_psutil_common.o
    unable to execute 'x86_64-conda_cos6-linux-gnu-gcc': No such file or directory
    error: command 'x86_64-conda_cos6-linux-gnu-gcc' failed with exit status 1
    
    ----------------------------------------
Command "/opt/conda/envs/renku/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-6ofroife/psutil/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-8fkhylnv/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-install-6ofroife/psutil/
The command '/bin/sh -c conda create -y -n renku python=3.6 &&     $CONDA_DIR/envs/renku/bin/pip install --no-cache-dir /wheels/renku*.whl &&     conda clean --yes --all &&     mkdir -p /home/$NB_USER/.renku/bin &&     ln -s $CONDA_DIR/envs/renku/bin/renku /home/$NB_USER/.renku/bin/renku &&     echo "export PATH=~/.renku/bin:$PATH" >> /home/$NB_USER/.bashrc' returned a non-zero code: 1
make: *** [base] Error 1
The command "make all" exited with 2.
```

A similar error creeps up when running `renku update`.

**Comment by rokroskar** [comment]
Actually in the case of `renku update` the error is different but potentially related: 

```
ImportError: libc.musl-x86_64.so.1: cannot open shared object file: No such file or directory
```

**Comment by rokroskar** [comment]
closed by #15
